### ISSUE TITLE
Address http://strato-dev4.blockapps.net doesn't exist anymore

### ISSUE THREAD

**Issue by dbblackdiamond** [blockapps/bloc#198]
Hi,

I have been following the "Getting Started" guide shown here: https://github.com/blockapps/strato-getting-started. I have run the `stratos-run.sh` script and everything worked fine. I then went to this guide http://developers.blockapps.net/dashboard/quick-starts/bloc-keyserver and followed the instructions on initializing my app and getting the bloc server running. 

Unfortunately, during the `bloc genkey` step, I got an error message saying: 

> Unhandled rejection Error: faucet: HTTPQuery: connect ECONNREFUSED 13.90.102.43:80

When I tried to access the URL directly, it doesn't seem to find it. Any idea on this?

Thanks a lot,
Bertrand.

**Comment by jcrain** [comment]
Hi Bertrand,

If you are following the getting started you should try pointing the bloc to your local installation. Try replacing the `strato-dev4 url` with `http://localhost/strato-api`

Thanks!

**Comment by dbblackdiamond** [comment]
Ok.... I thought it was something like that, but wasn't sure. Thanks a lot for letting me know. I will give this a try and let you know how it goes.

**Comment by dbblackdiamond** [comment]
@jcain: I have made some progress thanks to you. My issue right now though is that every time I try to do something using curl, I get the following error message: 

> {"errorTags":["HTTPQuery","submitTransaction","Transaction"],"message":"connect ECONNREFUSED 192.168.56.30:80"}

And yet, it seems like the action is still being performed. I have tried to create a new user and it gave me that error message and yet my user got properly created. My bloc server is running on host 192.168.56.30. Any idea?
